invenio_app_ils/views.py

```diff
@@ -56,6 +54,12 @@ def index(path):
     return render_template("invenio_app_ils/main.html")
 
 
+@main_blueprint.route('/ping', methods=['HEAD', 'GET'])
```

Comment:
Because you have done the testing part already, I would change the line 53 with the below
```
@main_blueprint.route("/", methods=["GET"])
@main_blueprint.route("/<path:path>", methods=["GET"])
```
as otherwise the route to slash is giving an error.